entgql/template/node.tmpl

```diff
@@ -15,13 +15,13 @@ in the LICENSE file in the root directory of this source tree.
 {{ end }}
 
 {{ $gqlNodes := filterNodes $.Nodes }}
-{{ $idType := findIDType $gqlNodes $.IDType }}
+{{ $v := assertIDTypes $gqlNodes $.IDType }}
```

Comment:
is there a better way to handle this? `assertIDTypes` could just return an error but then it outputs a `<nil>` into the generate code